LibAddonMenu-2.0/LibAddonMenu-2.0/LibAddonMenu-2.0.lua

```diff
@@ -78,10 +78,73 @@ local function GetStringFromValue(value)
     return value
 end
 
+local FAQ_ICON_COLOR = ZO_ColorDef:New("FFFFFF") -- white
+local FAQ_ICON_MOUSE_OVER_COLOR = ZO_ColorDef:New("B8B8D3") -- dark-blue/white
+
 local function GetColorForState(disabled)
     return disabled and ZO_DEFAULT_DISABLED_COLOR or ZO_DEFAULT_ENABLED_COLOR
 end
 
+local function CreateFAQTexture(parent, controlData, controlName, anchorPoint, anchorRelativeControl, anchorRelativePoint, anchorX, anchorY)
+    anchorPoint = anchorPoint or TOPLEFT
+    anchorRelativeControl = anchorRelativeControl or parent
+    anchorRelativePoint = anchorRelativePoint or TOPLEFT
+    anchorX = anchorX or 0
+    anchorY = anchorY or 0
+    local onMouseExitAlpha = 0.4
+
+    local helpUrl = controlData and controlData.helpUrl and GetStringFromValue(controlData.helpUrl)
+    if not helpUrl or helpUrl == "" then return end
+    local faqControl = wm:CreateControl(nil, parent, CT_TEXTURE)
+    faqControl:SetHidden(false)
+    faqControl:ClearAnchors()
+    faqControl:SetDrawLayer(DL_OVERLAY)
+    faqControl:SetTexture("\\ESOUI\\ART\\miscellaneous\\help_icon.dds") --small (i): \\ESOUI\\ART\\help\\help_tabicon_tutorial_up.dds
+    if controlData.isHalfWidth then
+        faqControl:SetDimensions(21, 21)
+    else
+        faqControl:SetDimensions(23, 23)
+    end
+
+    faqControl:SetColor(FAQ_ICON_COLOR:UnpackRGBA())
+    faqControl:SetAlpha(onMouseExitAlpha)
+    faqControl:SetAnchor(anchorPoint, anchorRelativeControl, anchorRelativePoint, anchorX, anchorY-1.5)
+
+    faqControl.isMouseOver = false
+    faqControl.data = faqControl.data or {}
+    faqControl.data.helpUrl = helpUrl
+    faqControl.data.tooltipText = util.L.WEBSITE .. ": " .. helpUrl
```

Comment:
Maybe use string.format with a template defined in a constant variable. e.g. HELP_URL_TOOLTIP:format(...)

Reply:
I'm not sure if this would be an overkill for a simple string concatenation with 2 variables...